contracts/interfaces/IModuleFactory.sol

```diff
@@ -71,6 +72,14 @@ interface IModuleFactory {
      */
     function changeUsageCost(uint256 _newUsageCost) external;
 
+    /**
+     * @notice Used to change the currency and amount of usage and setup cost
+     * @param _setupCost new setup cost
+     * @param _usageCost new usage cost
+     * @param _isCostInPoly new usage cost currency. USD or POLY
+     */
+    function changeCostsAndType(uint256 _setupCost, uint256 _usageCost, bool _isCostInPoly) external;
```

Comment:
can we use a similar name as present in the STR contract like `changeCostsAmountAndCurrency()` for consistency?

Reply:
Hmm. I tried to keep this consistent with the naming module factory. We call them costs and types instead of fees and currency in the module factory.